homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -365,33 +367,53 @@ async def async_step_init(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Manage the options."""
+
+        values = user_input or self.config_entry.options
+        schema = vol.Schema(
+            {
+                vol.Optional(
+                    CONF_DISABLE_RTSP,
+                    description={
+                        "suggested_value": values.get(CONF_DISABLE_RTSP, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_ALL_UPDATES,
+                    description={
+                        "suggested_value": values.get(CONF_ALL_UPDATES, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_OVERRIDE_CHOST,
+                    description={
+                        "suggested_value": values.get(CONF_OVERRIDE_CHOST, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_MAX_MEDIA,
+                    description={
+                        "suggested_value": values.get(CONF_MAX_MEDIA, DEFAULT_MAX_MEDIA)
+                    },
+                ): vol.All(vol.Coerce(int), vol.Range(min=100, max=10000)),
+                vol.Optional(
+                    CONF_IGNORED,
+                    description={"suggested_value": values.get(CONF_IGNORED, "")},
+                ): str,
```

Comment:
I just realized from reviewing the documentation PR, that this is actually a comma-separated list hidden in a single text field.

I think this should not be allowed or merged (and maybe reverted).

Instead, lets use a multi-select, allowing users to uncheck items from a list (instead of having to deal with unfriendly string).